beacon_node/http_api/src/block_id.rs

```diff
@@ -296,17 +296,24 @@ impl BlockId {
                     ))
                 })?
         } else {
-            BlobSidecarList::default()
+            BlobSidecarList::empty_uninitialized()
```

Comment:
`max_len` is not specified here - i think it's not an issue since it's not encoded when `BlobSidecarList` is used as a standalone object